src/ansible.cfg

```diff
@@ -2,3 +2,4 @@
 host_key_checking = False
 roles_path = ./roles
 callback_result_format = yaml
+log_path=/var/log/foremanctl/foremanctl.log
```

Comment:
Should this default to a relative path, `.var` and then be overriden in the RPM? Perhaps this should be an Obsah controlled element similar to how state storage( e.g. parameters) are handled.

Reply:
> Should this default to a relative path

I was thinking about it, but then I thought: what if the user runs `foremanctl` from a different directory?

The user can be in `~/` and for a different run, in the `/` directory, or somewhere else entirely. 

> Perhaps this should be an Obsah

I haven't thought about that one, but that could be the way as well.